lms/templates/dashboard/course.underscore

```diff
@@ -0,0 +1,643 @@
+<% if (!!settings.enable_verified_certificates) {
+
+        var modeClass = model.course_verified_certs.display_mode;
+
+        if (modeClass != ''){
+            modeClass = ' ' + modeClass ;
+        } else {
+            modeClass = '';
+        }
+    }
+    var courseTarget = _.sprintf( '/courses/%(courseKey)s/info', { courseKey: model.course_key });
+    var courseHomePageText = _.sprintf('%(courseNumber)s %(courseName)s Home Page', { courseNumber: model.course_number, courseName: model.course_name});
+    var courseCoverImageText = _.sprintf('%(courseNumber)s %(courseName)s Cover Image', { courseNumber: model.course_number, courseName: model.course_name});
+    var dashboard_index = 0;
```

Comment:
What is this used for? It appears to always be set to 0. It should be defined in the View if it is actually needed.